### PR TITLE
Update `get_ready_prs` function

### PR DESCRIPTION
**Pull request by phil-r**
- makes it more "flat" 
- more tests! 🎉 
- adds `server/voters.json` to `.gitignore`

### CODE REVIEW

File: github_api/prs.py [hunk 1]
```diff
@@ -172,27 +172,30 @@ def get_ready_prs(api, urn, window):
...
+        if mergeable is True:
+            label_pr(api, urn, pr_num, [])
+            yield pr
+        elif mergeable is False:
+            label_pr(api, urn, pr_num, ["conflicts"])
+            if delta >= 60 * 60 * settings.PR_STALE_HOURS:
+                comments.leave_stale_comment(
+                    api, urn, pr["number"], round(delta / 60 / 60))
+                close_pr(api, urn, pr)
+                # mergeable can also be None, in which case we just skip it for now
```
Comment (PlasmaPower) [inline github_api/prs.py]: This comment should be unindented to the mergeable if statement, or moved to right before the mergeable if statement.

**Comment by PlasmaPower** [comment]
~Other than that small comment,~ :+1:

File: github_api/prs.py (hunk 1 above)
Comment (phil-r) [reply github_api/prs.py]: @PlasmaPower fixed!

**Review by WebMasta — COMMENTED** [review]
int main(){
   Cout << "Hello world";
   Return 0;
}

File: github_api/prs.py [hunk 2]
```diff
@@ -172,31 +172,32 @@ def get_ready_prs(api, urn, window):
             continue
 
         delta = (now - updated).total_seconds()
-
         is_wip = "WIP" in pr["title"]
 
-        # this is computed but unused.  there are issues with travis status not
+        # this is unused righ now.  there are issues with travis status not
```
Comment (PlasmaPower) [inline github_api/prs.py]: *right now

Comment (phil-r) [reply github_api/prs.py]: 🤦‍♂️ another timer reset

**Pull request merged**